_posts/2020-09-14-notes-on-building-python-for-windows-using-rbm.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Notes on building Python for Windows using RBM"
+author: yanmaani
+tags: [News]
+---
+
+Namecoin would like for the Tor Browser to support .bit domains.
+The only mature, lightweight way to do this is to use the wallet qua name resolver Electrum-NMC.
```

Comment:
Is `qua` a typo?  I don't see any reason to mention a wallet here; Electrum-NMC's wallet functionality is stripped in Tor Browser; only the name resolution functionality is relevant.

Reply:
To me, the word "wallet" has two meanings:
1) The thing that has your coins you interface with as a user
2) The user-facing thing that speaks the protocol

A web wallet is a wallet<sub>1</sub> but not a wallet<sub>2</sub>. The stripped Electrum-NMC is a wallet<sub>2</sub> but not a wallet<sub>1</sub>.

We use Electrum-NMC, which is a wallet, as a name resolver. We do not use it as a wallet.

Is there another word for "Bitcoin protocol speaking client"?